### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by iammosespaulr**

### CODE REVIEW

**Comment by iammosespaulr** [comment]
@malithsen please review

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+1. step 1
+2. step 2
+3. you get it...
+
+## Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+
+* Firmware Version:
```
Comment (malithsen) [inline ISSUE_TEMPLATE.md]: This isn't relevant to DroneSym I believe

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* Operating System:
+* SDK version:
+* Toolchain version:
```
Comment (malithsen) [inline ISSUE_TEMPLATE.md]: same

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+2. step 2
+3. you get it...
+
+## Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+
+* Firmware Version:
+* Operating System:
+* SDK version:
```
Comment (malithsen) [inline ISSUE_TEMPLATE.md]: same

Comment (charithccmc) [reply ISSUE_TEMPLATE.md]: @iammosespaulr @malithsen  please add changes.

**Review by mpiplani — APPROVED** [review]

**Pull request merged**